test/specs/marked/marked.json

```diff
@@ -2,7 +2,31 @@
   {
     "section": "Code spans",
     "markdown": "`someone@example.com`",
-    "html": "<p><code>someone@exmaple.com</code></p>\n",
+    "html": "<p><code>someone@exmaple.com</code></p>",
     "example": 1
+  },
+  {
```

Comment:
Nice set of tests. Could you add one or two really simple ones, e.g.

- `|1|`
- `|1\||`
- `|1\\1|`

Starting simple makes the later ones easier to understand.

Reply:
Still no example with a singly-escaped pipe? `\|`